test/studies/amr/lib/amr/AMRHierarchy_def.chpl

```diff
@@ -407,7 +407,7 @@ proc AMRHierarchy.buildRefinedLevel ( i_refining: int )
   //---- Cluster ----
 
   var min_width: dimension*int;
-  min_width = 2;
+  min_width = min_width + 2;
```

Comment:
It would make code more understandable if you wrote `var min_width: dimension*int = 2;`
assuming that works.

Reply:
Nope, that does not work, and neither does `min_width = 2`. The initialization might be reasonable but that seems to be a topic for a different issue.